### PR TITLE
Added getting help page

### PR DESCRIPTION
**Pull request by paolap**
Added getting help page from old wiki with a few minor changes

### CODE REVIEW

File: Wiki/services/services-help.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# CMS help
+
+The CMS team supports CLEX members with computational issues: 
+
++ **Running Models**: we support several models (see navigation bar on the left) and assist with others, subject to limitations of scope and team resources. 
++ **Data Analysis**: we assist with analysis and general computing tasks on NCI machines, and offer only limited support on other sites or home machines as we possess neither control over nor familiarity with such systems.
++ **Data Access and publishing**: we offer assistance in publishing data and accessing climate data available at NCI.
+The CMS team can be contacted for support through:
```
Comment (rkutteh) [inline Wiki/services/services-help.md:8]: Needs a line break before "The CMS team can be contacted..."

File: Wiki/services/services-help.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# CMS help
+
+The CMS team supports CLEX members with computational issues: 
+
++ **Running Models**: we support several models (see navigation bar on the left) and assist with others, subject to limitations of scope and team resources.
```
Comment (rkutteh) [inline Wiki/services/services-help.md:5]: Not sure if you want to keep "see navigation bar on the left". I guess it depends on the final format.

**Review by rkutteh — CHANGES_REQUESTED** [review]
I made a couple of comments in-text that you may want to check before merging.

File: Wiki/services/services-help.md (hunk 2 above)
Comment (paolap) [reply Wiki/services/services-help.md:5]: thanks, let's remove it for the moment, good spotting!

**Review by paolap — COMMENTED** [review]
I updated the files as requested and also added a warning box to highlight the last sentence:

 If any of the above expectations are not being met, please contact the CMS team.

**Review by rkutteh — APPROVED** [review]
Looks good.

**Pull request merged**